Gruntfile.js

```diff
@@ -239,7 +240,7 @@ module.exports = function(grunt) {
 
     githooks: {
       all: {
-        'pre-commit': 'jshint cover'
+        'pre-commit': 'jshint'
```

Comment:
Actually we do want this here, because there is no grunt cover task anymore. It will error out before the commit for that reason.